lib/std/os/linux/io_uring.zig

```diff
@@ -371,10 +371,10 @@ pub const IO_Uring = struct {
         },
     };
 
-    /// Queues (but does not submit) an SQE to perform a `read(2)` or `preadv` depending on the buffer type.
+    /// Queues (but does not submit) an SQE to perform a `read(2)` or `preadv(2)` depending on the buffer type.
     /// * Reading into a `ReadBuffer.buffer` uses `read(2)`
     /// * Reading into a `ReadBuffer.iovecs` uses `preadv(2)`
-    ///   If you want to do a `preadv2()` then set `rw_flags` on the returned SQE. See https://linux.die.net/man/2/preadv.
+    ///   If you want to do a `preadv2(2)` then set `rw_flags` on the returned SQE. See https://linux.die.net/man/2/preadv.
```

Comment:
The link https://linux.die.net/man/2/preadv doesn't actually document `preadv2`.
We should probably update it to a link that does.

Reply:
Good catch! I went with [this](https://man7.org/linux/man-pages/man2/preadv2.2.html), is that alright?